gammapy/analysis/core.py

```diff
@@ -172,7 +172,7 @@ def read_models(self, path):
         models = Models.read(path)
         self.set_models(models)
 
-    def run_fit(self, optimize_opts=None):
+    def run_fit(self):
```

Comment:
Here it is no longer possible options on the `Fit` class. Does it mean we need to update the configuration to introduce parameters on `FitConfig`?

Reply:
This would be the perfect solution, however modifying the `Fit` class in place always works:
```python3
analysis.fit.backend = "minuit"
analysis.fit.optimize_opts = {"something":  2}
```

However we should probably introduce a `Fit.__str__` so that it become easy to see which options are set.